images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -49,6 +49,11 @@ function Get-ClangFormatVersions {
     return "Clang-format " + $clangFormatVersions
 }
 
+function Get-ClangTidyVersions {
+    $clangFormatVersions = Get-ClangToolVersions -ToolName "clang-tidy"
+    return "Clang-tidy " + $clangFormatVersions
```

Comment:
nitpick: `return "Clang-tidy " + $clangFormatVersions` -> `return "Clang-tidy $clangFormatVersions"`

Reply:
I was being consistent with the other `Get-Clang*` functions above here. I could update them as well?

Reply:
yes, please feel free to update them as well

Reply:
Cool, done 👍